tests/fe/fe_rannacher_turek_01.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2015 by the deal.II authors
+// Copyright (C) 2015, 2017 by the deal.II authors
```

Comment:
This should be hyphenated: regardless the update copyright script will do this for us.